app/controllers/support_interface/providers_controller.rb

```diff
@@ -0,0 +1,15 @@
+module SupportInterface
+  class ProvidersController < SupportInterfaceController
+    def index
+      @providers = Provider.all
+    end
+
+    def sync
+      Rails.configuration.providers_to_sync[:codes].each do |code|
+        SyncProviderFromFind.call(provider_code: code)
+      end
```

Comment:
(I know you like this here, but this really sound like a service :) )

Reply:
I prefer this interface as there is less reason for the service to change, and it can be easily reused by other callers